robustirc/src/jepsen/robustirc.clj

```diff
@@ -0,0 +1,230 @@
+(ns jepsen.robustirc
+  (:require
+    [clojure.tools.logging    :refer [debug info warn]]
+    [clojure.java.io          :as io]
+    [clojure.string           :as str]
+    [clojure.core.async       :as async]
+    [clj-http.client :as httpclient]
+    [digest :as digest]
+    [jepsen
+      [client :as client]
+      [core :as jepsen]
+      [db :as db]
+      [tests :as tests]
+      [control :as c :refer [|]]
+      [checker :as checker]
+      [nemesis :as nemesis]
+      [generator :as gen]]
+    [jepsen.os.debian :as debian]
+    [cheshire.core            :as json])
+  (:import
+    [java.net URL]))
+
+(defn db [version]
```

Comment:
Doesn't matter to me--I just parameterize it so I can test multiple versions easily, and it makes the test more reproducible because we target a specific version and not the latest. :)